### PR TITLE
Add delete session command

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

**Review by raydenlim — COMMENTED** [review]
LGTM, no scary errors detected

File: src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Deletes a session identified using it's displayed index from the session list book.
+ */
+public class DeleteSessionCommand extends Command {
+    public static final String COMMAND_WORD = "deletesession";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the session identified by the session number.\n"
+            + "Parameters: "
+            + PREFIX_SESSION + "SESSION_NUMBER (session must already be created) "
```
Comment (raydenlim) [inline src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java]: might want to add a \n after paremeters.
will look cleaner on the UI

**Review by raydenlim — APPROVED** [review]
:) LGTM

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -228,7 +229,14 @@ public void parseCommand_updateSessionRemark() throws Exception {
         assertTrue(parser.parseCommand(UpdateSessionRemarkCommand.COMMAND_WORD
                 + whiteSpace + PREFIX_SESSION + sessionNumber
                 + whiteSpace + PREFIX_SESSION_REMARK + sessionRemark) instanceof UpdateSessionRemarkCommand);
+    }
 
+    @Test
```
Comment (raydenlim) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:233]: hmm, i see alot of the same `String whiteSpace = " "` being used. Might want to change it to a global constant?

Comment (ldinghan) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java:233]: Sounds good! Added the final String for this file and replaced the " " strings within each test to use `WHITESPACE` instead

File: src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java (hunk 1 above)
Comment (ldinghan) [reply src/main/java/seedu/address/logic/commands/DeleteSessionCommand.java]: will do

**Pull request merged**